.github/workflows/sql-test.yml

```diff
@@ -16,9 +43,9 @@ jobs:
           - os: linux
             arch: amd64
             runner: ubuntu-latest
-          - os: windows
-            arch: amd64
-            runner: windows-latest
+#          - os: windows todo see sql-release.yml
+#            arch: amd64
+#            runner: windows-latest
```

Comment:
Same comment about adding back as in `sql-release`?

Reply:
fixed